parser/parser.y

```diff
@@ -3437,6 +3437,11 @@ Statement:
 |	UnionStmt
 |	UpdateStmt
 |	UseStmt
+|	SubSelect
```

Comment:
I tried to add '(' SelectStmt ')' into SelectStmt rule, but I got unresolveable confliction.